src/main/java/org/opensearch/knn/index/SpaceType.java

```diff
@@ -105,4 +107,23 @@ public static SpaceType getSpace(String spaceTypeName) {
         }
         throw new IllegalArgumentException("Unable to find space: " + spaceTypeName);
     }
+
+    public static VectorSimilarityFunction spaceTypeToVectorSimilarityFunction(SpaceType spaceType) {
```

Comment:
shouldn't this be a function of enums rather than a public static func?

Reply:
Yes you are correct. Fixed